package.json

```diff
@@ -52,7 +52,7 @@
     "rename-package": "node -r ./scripts/ts-node-register ./scripts/rename-package.ts",
     "run:published": "node ./scripts/monorepo/runPublished.js",
     "runto:lerna": "node ./scripts/monorepo/runTo.js",
-    "satisfied": "satisfied --skip-invalid --ignore \"prettier|angular|lit|sass|@storybook/web-components|svelte|@testing-library|vue\"",
+    "satisfied": "satisfied --skip-invalid --ignore \"prettier|angular|lit|sass|@storybook/web-components|@storybook/html|svelte|@testing-library|vue\"",
```

Comment:
This is because one of the core storybook packages lists `@storybook/html` as a peer dep (even though it doesn't appear to actually be needed). We do have `@storybook/html` in our repo for web-components, but it MUST NOT be hoisted to the root because both `@storybook/react` and `@storybook/html` define `build-storybook`/`start-storybook` in their `bin`, and it's nondeterministic which one yarn will actually choose to link under the root `node_modules/.bin`.